### ISSUE TITLE
long and short hull OHP class FFG 

### ISSUE THREAD

**Issue by trevor99999** [PygmalionOfCyprus/cmo-db-requests#3909]
### DB Selector

Both

### Baseline

#742

### Hypothetical

No

### Name

Oliver Hazard Perry

### Class

Oliver Hazard Perry

### Country

USA

### Service

Navy

### In Commission

1977-1997

### Number in Service

Short hull = 25x

### Length (m)

124

### Beam (m)

14

### Draft (m)

6.7

### Standard Displacement (t)

3500

### Full Displacement (t)

4200

### Empty Displacement (t)

2800

### Complement

176

### Primary Propulsion Type

Gas Turbine

### Propulsion

2 × General Electric LM2500-30 gas turbines

### Flank Speed (kt)

30

### Range (nm) at Speed (kt)

4500 nmi @ 20 kts

### Sensors

AN/SPS-49, 
AN/SPS-55, 
Mk 92 fire-control system
Sonar: SQS-56, SQR-19 Towed Array
AN/SLQ-32(V)2, Flight III with sidekick,
Mark 36 SRBOC
AN/SLQ-25 Nixie

### Mounts

1× single-arm Mk 13 Missile Launcher w/ 40x magazine of SM-1MR/Harpoon
Removed from the U.S. Navy ships starting in 2003 on retirement of the SM-1 missile.
Mk 38 Mod 2 Machine Gun Systems installed on platforms over the removed MK 13 launchers
2 × triple Mark 32 Anti-submarine warfare torpedo tubes with Mark 46 or Mark 50 anti-submarine warfare torpedo (24 torpedoes)
1 × OTO Melara 76 mm/62 caliber naval gun
1 × 20 mm Phalanx CIWS

### Aircraft & Docking Facilities

2× LAMPS I (SH-2 Seasprite) on the short-hulled ships
2x LAMPS III (SH-60 Seahawk) on the long-hulled ships

### Ship Codes

Nuclear Shock Resistant, Degaussed Steel Hull, Prairie Masker

### Comments

OHP FFG-7, -9 to -27,-30, -31, -34 short hull w/ 2x LAMPS I SH-2 helo cap
OHP FFG-28, -29, -32, -33, -35 to -61 long hull w/2x LAMPS III SH-60 help cap

### Sources

https://en.wikipedia.org/wiki/Oliver_Hazard_Perry-class_frigate
https://www.globalsecurity.org/military/systems/ship/ffg-7.htm
https://www.nvr.navy.mil/SHIPDETAILS/SHIPSDETAIL_FFG_7.HTML
http://www.maritimequest.com/warship_directory/us_navy_pages/frigates/pages/oliver_hazard_perry_page_1.htm

Personal Library
Naval Institute Guide to Combat Fleets of the World 2002-2003, pp963-966
Jane's Fighting Ships 1991-92, pp748-749

**Comment by KLABDB3K** [comment]
There are literally multiple versions of practically every iteration of OHP in the DB and have been from the outset? Don't understand what I've missed? Have you looked under the various ship names which where the lead ship for each of the blocks?

**Comment by trevor99999** [comment]
This is interesting.
Earlier today, all the OHP class FFG variants (1978, 81, 84, 86, 91, 94, 98, 2001-2003) DB #'s 1162, 2286, 1163, 1165, 726, 1164, 1547, 1768, 1769 in DB3000 v502 had 2x medium helicopter capacity. I specifically checked to see if there was any difference between them before I filed a request. Now they all say 2x small aircraft.
So now none of them can carry a SH-60 Seahawk, which is a medium helicopter.
FFG-7, 9-27 inclusive, 30,31,34 were short hulls, which carried the SH-2 Seasprite (small helo), and apparently could/did not carry the SH-60 Seahawk.
FFG-8, 28, 29, 32, 33, 35-61 inclusive were long hulls that carried (or could carry) the SH-60 Seahawk.(medium helo)
 https://en.wikipedia.org/wiki/Oliver_Hazard_Perry-class_frigate#List_of_vessels
Regards





    On Thursday, October 19, 2023 at 06:14:26 p.m. EDT, KLABDB3K ***@***.***> wrote:  
 
 


There are literally multiple versions of practically every iteration of OHP in the DB and have been from the outset? Don't understand what I've missed? Have you looked under the various ship names which where the lead ship for each of the blocks?

**Comment by FrangibleCover** [comment]
The vessels categorised under "FFG 36 Underwood [Perry Class]" are the long hull OHPs, carrying two medium helicopters for the LAMPS III Seahawk capability. They begin with Ship 742, which is the ship you have requested a copy from, so I think earlier you looked at the Underwood and now you're looking at the Perry. Both types look fine to me in the DB.

**Comment by bostonmyk** [comment]
The current builds are fine. The pennant number matched with the type in the comments would be helpful though (like what is done with the Knox class names).

**Comment by FrangibleCover** [comment]
Agreed, and I'm also not sure why FFG-36 Underwood is the subclass leader when FFG-8 McInerney was the first long hull type ordered, built and commissioned. Am I missing something about her?

**Comment by tbrandon517** [comment]
> Agreed, and I'm also not sure why FFG-36 Underwood is the subclass leader when FFG-8 McInerney was the first long hull type ordered, built and commissioned. Am I missing something about her?

The FFG 8, 29, 32, and 33 were built as "short-hull" warships but were later modified into "long-hull" warships. FFG-36 onwards were long-hulls from the beginning.

**Comment by KLABDB3K** [comment]
Maybe the request could be for some notes to be added so you its doesn't need OHP expertise to add them to scenarios?.

**Comment by tbrandon517** [comment]
> Maybe the request could be for some notes to be added so there is an at a glance reminder in scenarios about the hangar size?.
> 
> Underwood (Perry - 2 x SH-60 Hangar) ?? Perry (OHP - 2 X SH-2 Hangar) ??

Or indicate FFG 7 entries as Short-Hull and FFG-36 as Long-Hull?

**Comment by trevor99999** [comment]
> > Maybe the request could be for some notes to be added so there is an at a glance reminder in scenarios about the hangar size?.
> > Underwood (Perry - 2 x SH-60 Hangar) ?? Perry (OHP - 2 X SH-2 Hangar) ??
> 
> Or indicate FFG 7 entries as Short-Hull and FFG-36 as Long-Hull?

I agree. As it stands, now _**no**_ OHP FFG can carry an SH-60. Only SH-2's. It would only require a separate Short/Long Hull entry for each year that the only difference would be the size of the helo that could be carried.
As well the ships that were exported would also need to be updated in their respective navy listings

**Comment by FrangibleCover** [comment]
> I agree. As it stands, now _**no**_ OHP FFG can carry an SH-60. Only SH-2's. It would only require a separate Short/Long Hull entry for each year that the only difference would be the size of the helo that could be carried. As well the ships that were exported would also need to be updated in their respective navy listings

As has been indicated, the Perrys labelled FFG-36 Underwood (beginning DB3k DBID 742) are the Long Hull variants that are perfectly capable of carrying two SH-60s exactly as they should be. All subsequent discussion has been about whether the labelling of the Underwood subclass is sufficient, which I think we have now demonstrated it is not.

**Comment by trevor99999** [comment]
> > I agree. As it stands, now _**no**_ OHP FFG can carry an SH-60. Only SH-2's. It would only require a separate Short/Long Hull entry for each year that the only difference would be the size of the helo that could be carried. As well the ships that were exported would also need to be updated in their respective navy listings
> 
> As has been indicated, the Perrys labelled FFG-36 Underwood (beginning DB3k DBID 742) are the Long Hull variants that are perfectly capable of carrying two SH-60s exactly as they should be. All subsequent discussion has been about whether the labelling of the Underwood subclass is sufficient, which I think we have now demonstrated it is not.

The p[lot thickens. All year variants were originally in the DB as able to carry medium helos, which is where the issue began. Then a couple of days ago they all changed to small helos. Now they're back to medium.
The short hull OHP's were unable to carry the SH-60 (medium helo), only the SH-2 (small helo).

**Comment by FrangibleCover** [comment]
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/5f38bdd4-25e0-42d2-a732-63a20586d491)
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/9b83ea7f-5a99-4e75-8f7f-ddd6efa0ba52)

There are already both types. Here they are. Note that one is called Underwood and one is called Perry, but they are both denoted as Perry class in the square brackets. You can check this for yourself, look for ID 742 and ID 1162 _specifically_ to ensure that you don't make the same mistake a fourth time. The database has not changed and cannot change, Version 502 is fixed and set in stone forever, all new changes will be made to Version 503.

**Comment by tbrandon517** [comment]
Going through all the Perrys in DB 502. All Helo facilities seems correct. Do think adding a Long-Hull/Short-Hull tag in the unit name would be useful on a quick pass.

**Comment by PygmalionOfCyprus** [comment]
Names clarified in 503.
